metrics-core/src/main/java/software/amazon/swage/metrics/record/cloudwatch/CloudWatchRecorder.java

```diff
@@ -388,6 +391,11 @@ protected void record(
             return;
         }
 
+        if (!isValid(label)) {
+            log.warn("Invalid metric name: '" + label + "'");
```

Comment:
nonblocking: I'm not sure I'm decided one way or another on this, but would it be reasonable to emit the rest of/more of the metric information (value, unit, timestamp)?